yarn-project/end-to-end/src/e2e_persistence.test.ts

```diff
@@ -64,18 +64,15 @@ describe('Aztec persistence', () => {
     ownerAddress = ownerWallet.getCompleteAddress();
     ownerSalt = ownerWallet.salt;
 
-    const deployer = TokenContract.deploy(ownerWallet, ownerWallet.getAddress(), 'Test token', 'TEST', 2);
-    await deployer.simulate({});
-
-    const contract = await deployer.send().deployed();
+    const contract = await TokenContract.deploy(ownerWallet, ownerWallet.getAddress(), 'Test token', 'TEST', 2)
+      .send()
+      .deployed();
```

Comment:
Alex forgot this simulate() here so I nuked it here and in the other 2 occurrences bellow.
<img width="568" alt="image" src="https://github.com/AztecProtocol/aztec-packages/assets/13470840/67a58eeb-9393-4a8d-96ea-6813a5c13ef6">